### PR TITLE
DOC: Add Crop Volume module documentation

### PR DESCRIPTION
**Pull request by RafaelPalomar**

### CODE REVIEW

**Comment by RafaelPalomar** [comment]
The image associated to the module can be found here: 

https://www.dropbox.com/scl/fo/1p1alfz5vsxooasri6o8j/h?dl=0&rlkey=pd62bblm6206oetmsfmpwm1sp](https://www.dropbox.com/scl/fo/1p1alfz5vsxooasri6o8j/h?dl=0&rlkey=pd62bblm6206oetmsfmpwm1sp) and it is expected to be uploaded through github release on the following url:

https://github.com/Slicer/Slicer/releases/download/docs-resources/module_cropvolume.png

@lassoan, @jcfr could you upload the image, I believe I don't have the rights to do that.

**Comment by jcfr** [comment]
Thanks @RafaelPalomar, image has been uploaded :heavy_check_mark: :rocket: 

Current resolution is `1920x1200`, while we may have to tweak the size, this should allow to move forward.

**Comment by jcfr** [comment]
Since the file `cropvolume.md` is in fact named `cropvolume.md ` (with an extra space at the end), I have to force push the branch.

Considering that force pushing is not authorized:

```
ERROR: Permission to ALive-research/Slicer.git denied to jcfr.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

I will be close this pull-request and create a new one.

**Comment by RafaelPalomar** [comment]
> Thanks @RafaelPalomar, image has been uploaded heavy_check_mark rocket
> 
> Current resolution is `1920x1200`, while we may have to tweak the size, this should allow to move forward.

@jcfr, should the image be reduced, you mean?

**Comment by jcfr** [comment]
> should the image be reduced, you mean?

Indeed, to match size already used in the dicom module, I re-scaled to ~ `1280x800` (using linear interpolation)

File: Docs/user_guide/modules/cropvolume.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+## Volume Information
+
+This section displays the spacing and dimensions of the input and output volume.
+
+  - **Input volume:** Spacing and Dimensions information about the input volume.
+
+  - **Output volume:** Spacing and Dimensions information about the input volume.
+
+# Contributors
```
Comment (jcfr) [inline Docs/user_guide/modules/cropvolume.md:71]:
```suggestion
## Contributors
```

File: Docs/user_guide/modules/cropvolume.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Andrey Fedorov (BWH, SPL) and Ron Kikinis (BWH, SPL).
+
+# Acknowledgments
```
Comment (jcfr) [inline Docs/user_guide/modules/cropvolume.md:75]:
```suggestion
## Acknowledgments
```

File: Docs/user_guide/modules/cropvolume.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+removal process takes place with the help of a region of interest (ROI).
+This region of interest can be manipulated either from the 3D View (possibly,
+with the help of volume rendering or 3D surface models) or from any of the Slice
+views (2D). In addition, the module
+includes advance functionality to fill the region outside the original volume,
+and control the spacing and the interpolation scheme used inside the cropped
+region.
+
+![](https://github.com/Slicer/Slicer/releases/download/docs-resources/module_cropvolume.png)
+
```
Comment (jcfr) [inline Docs/user_guide/modules/cropvolume.md:15]:
```suggestion
## Panels and their use
```

File: Docs/user_guide/modules/cropvolume.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  - **Fit to Volume:** This will resize the ROI to fit the extent of the Input volume specified.
+
+  - **Output volume:** The output volume that represents the result of the
+    cropping operation. This widget allows for the selection of an already
+    existing volume (e.g., the input volume itself) or the creation of a new
+    output volume node to store the results.
+
+After setting these parameters the user can click the Apply button to perform the operation. WARNING: The user should be careful when selecting the input and output volumes: selecting the same output volume as the input may render the input volume unusable for other operations.
+
+## Advanced parameters
```
Comment (jcfr) [inline Docs/user_guide/modules/cropvolume.md:41]:
```suggestion
### Advanced parameters
```

File: Docs/user_guide/modules/cropvolume.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    value is smaller than 1.0, then the cropped volume will have higher
+    resolution than the input volume.
+
+    - **Isotropic spacing:** Enable/disable isotropic spacing. When this is
+     enable, the voxel spacing of the output volume will be set to the lowest
+     spacing in any of the axes present in the input volume.
+
+    - **Interpolator:** Type of interpolation used: Nearest neighbor, Linear, Windowed Sinc, B-spline.
+
+## Volume Information
```
Comment (jcfr) [inline Docs/user_guide/modules/cropvolume.md:63]:
```suggestion
### Volume Information
```

File: Docs/user_guide/modules/cropvolume.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+This region of interest can be manipulated either from the 3D View (possibly,
+with the help of volume rendering or 3D surface models) or from any of the Slice
+views (2D). In addition, the module
+includes advance functionality to fill the region outside the original volume,
+and control the spacing and the interpolation scheme used inside the cropped
+region.
+
+![](https://github.com/Slicer/Slicer/releases/download/docs-resources/module_cropvolume.png)
+
+## Basic parameters
```
Comment (jcfr) [inline Docs/user_guide/modules/cropvolume.md:16]:
```suggestion
### IO
```

**Comment by jcfr** [comment]
Closing. This is superseded by https://github.com/Slicer/Slicer/pull/6663

**Pull request closed without merging**